### PR TITLE
add github-guard plugin

### PR DESCRIPTION
**Pull request by onuryilmaz**

### CODE REVIEW

File: github-guard/plugindefinition.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+spec:
+  version: "0.1.0"
+  displayName: Github Guard
+  description: Manages Github teams, team memberships and repository & team assignments
+  docMarkDownUrl: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/github-guard/README.md
+  icon: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/github-guard/img/logo.png
+  helmChart:
+    name: github-guard
+    repository: oci://ghcr.io/cloudoperators/greenhouse-extensions/charts
+    version: 0.1.0
```
Comment (kengou) [inline github-guard/plugindefinition.yaml:17]:
```suggestion
    version: 0.1.0

```

File: github-guard/charts/templates/serviceaccount.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+kind: ServiceAccount
+metadata:
+  name: {{ include "github-guard-greenhouse.fullname" . }}-controller-manager
+  labels:
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: github-guard-greenhouse
+    app.kubernetes.io/part-of: github-guard-greenhouse
+  {{- include "github-guard-greenhouse.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.controllerManager.serviceAccount.annotations | nindent 4 }}
```
Comment (kengou) [inline github-guard/charts/templates/serviceaccount.yaml:16]:
```suggestion
    {{- toYaml .Values.controllerManager.serviceAccount.annotations | nindent 4 }}

```

File: github-guard/charts/templates/manager-rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    app.kubernetes.io/part-of: github-guard-greenhouse
+  {{- include "github-guard-greenhouse.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: '{{ include "github-guard-greenhouse.fullname" . }}-manager-role'
+subjects:
+- kind: ServiceAccount
+  name: '{{ include "github-guard-greenhouse.fullname" . }}-controller-manager'
+  namespace: '{{ .Release.Namespace }}'
```
Comment (kengou) [inline github-guard/charts/templates/manager-rbac.yaml:98]:
```suggestion
  namespace: '{{ .Release.Namespace }}'

```

File: github-guard/charts/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        resources: {{- toYaml .Values.controllerManager.manager.resources | nindent 10
+          }}
+        securityContext: {{- toYaml .Values.controllerManager.manager.containerSecurityContext
+          | nindent 10 }}
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: {{ include "github-guard-greenhouse.fullname" . }}-controller-manager
+      terminationGracePeriodSeconds: 10
```
Comment (kengou) [inline github-guard/charts/templates/deployment.yaml:59]:
```suggestion
      terminationGracePeriodSeconds: 10

```

**Review by kengou — COMMENTED** [review]
LGTM, I have some nitpick for the no new line in the end if the files

**Comment by onuryilmaz** [comment]
> LGTM, I have some nitpick for the no new line in the end if the files

merged those changes, thanks 👍

**Review by kengou — APPROVED** [review]

**Review by IvoGoman — APPROVED** [review]

**Pull request merged**